ompi/communicator/comm_cid.c

```diff
@@ -840,6 +840,7 @@ static int ompi_comm_allreduce_pmix_reduce_complete (ompi_comm_request_t *reques
     opal_pmix_pdata_t pdat;
     opal_buffer_t sbuf;
     int rc;
+    int bytesWritten;
```

Comment:
Minor nit: we don't typically use camelCase in Open MPI; we typically use multiple_words.  This is pretty trivial -- we would not reject a patch just for this reason, though -- I mention it just for "style fit" reasons.